modules/@apostrophecms/schema/ui/apos/components/AposInputSlug.vue

```diff
@@ -149,7 +149,7 @@ export default {
       }
       if (this.field.page) {
         const matches = slug.match(/[^/]+$/);
-        slug = matches[0] || '';
+        slug = (matches && matches[0]) || '';
```

Comment:
Unrelated console warning fix, came up when editing page settings for the home page.